python/ray/_raylet.pyx

```diff
@@ -1120,3 +1119,12 @@ cdef void async_retry_with_plasma_callback(shared_ptr[CRayObject] obj,
                 AsyncGetResponse(
                     plasma_fallback_id=ObjectID(object_id.Binary()),
                     result=None)))
+
+cdef void async_plasma_callback(CObjectID object_id,
```

Comment:
Can you put this above the `CoreWorker` definition (near the task execution callback)?

Reply:
Sure!